### PR TITLE
softdevice_controller: rev c5a8a5af402963933cbe3c33fe19f63ced0a2297

### PR DESCRIPTION
**Pull request by balaji-nordic**
CHANGELOG.rst contains the list of changes.

Signed-off-by: 'Balaji Srinivasan' <basr@nordicsemi.no>

### CODE REVIEW

**Comment by joerchan** [comment]
@balaji-nordic Looks OK, not sure what compliance is complaining about, but might be because it is a draft.

You will need a manifest update PR in sdk-nrf as well.

**Review by rugeGerritsen — APPROVED** [review]

File: softdevice_controller/CHANGELOG.rst [hunk 1]
```diff
@@ -9,18 +9,19 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
-Master branch
+v1.5-branch
```
Comment (ru-fu) [inline softdevice_controller/CHANGELOG.rst:12]:
```suggestion
nRF Connect SDK v1.5.1
```
I don't think it makes sense to mention the v1.5 branch here - we don't tell users to use version branches (only tags or master).
(Also add *** as needed in the next line.)

File: softdevice_controller/CHANGELOG.rst [hunk 2]
```diff
@@ -9,18 +9,19 @@ Changelog
...
 
 Added
 =====
 
 * Added radio front-end module (FEM) support, based on the :ref:`mpsl_fem` (DRGN-11059).
 
-Bug fixes
-=========
+Bugfixes
+========
```
Comment (ru-fu) [inline softdevice_controller/CHANGELOG.rst:21]:
```suggestion
Bug fixes
=========
```
Usage has changed.

**Review by ru-fu — CHANGES_REQUESTED** [review]

**Review by ru-fu — COMMENTED** [review]
Docs look good now, but need to squash the commits.

**Comment by balaji-nordic** [comment]
> @balaji-nordic Looks OK, not sure what compliance is complaining about, but might be because it is a draft.
> 
> You will need a manifest update PR in sdk-nrf as well.

https://github.com/nrfconnect/sdk-nrf/pull/4182 created for this.

**Comment by balaji-nordic** [comment]
> Docs look good now, but need to squash the commits.

Squashed the commits now.

**Review by ru-fu — APPROVED** [review]

**Review by joerchan — APPROVED** [review]

**Pull request merged**